probe_scraper/parsers/events.py

```diff
@@ -0,0 +1,84 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import third_party.parse_events
+
+
+def set_in_nested_dict(dictionary, path, value):
+    keys = path.split('/')
+    for k in keys[:-1]:
+        dictionary = dictionary[k]
+    dictionary[keys[-1]] = value
+
+
+def extract_events_data(e):
+    props = {
+        # source_field: target_field
+
+        # TODO: extract description.
+        "description": "description",
+        "expiry_version": "expiry_version",
+        "expiry_day": "expiry_day",
+        "cpp_guard": "cpp_guard",
+
+        "methods": "details/methods",
+        "objects": "details/objects",
+        "record_in_processes": "details/record_in_processes",
+        # TODO: extract key descriptions too.
+        "extra_keys": "details/extra_keys",
+    }
+
+    defaults = {
+        "expiry_version": "never",
+        "expiry_day": "never",
+        "name": e.methods[0],
+        "description": "<TODO>",
+        "cpp_guard": None,
+    }
+
+    data = {
```

Comment:
Is this helpful in style?
I find it more readable to keep data-oriented dicts in this same style, to quickly see the contents.